pkg/backup/backup/backup_manager.go

```diff
@@ -220,7 +220,8 @@ func (bm *backupManager) makeExportJob(backup *v1alpha1.Backup) (*batchv1.Job, s
 					VolumeMounts: []corev1.VolumeMount{
 						{Name: label.BackupJobLabelVal, MountPath: constants.BackupRootPath},
 					},
-					Env: envVars,
+					Env:       envVars,
```

Comment:
Please also update the `backup_cleaner.go` for clean jobs and `restore_manager.go` for restore jobs.